pkg/registry/core/persistentvolume/storage/storage_test.go

```diff
@@ -201,6 +204,65 @@ func TestUpdateStatus(t *testing.T) {
 	}
 }
 
+func TestUpdateStatusLastPhaseTransitionTime(t *testing.T) {
```

Comment:
I think it would be good to make this table driven so you can test the following scenarios:

- Feature gate enabled, phase changes
- Feature gate enabled, phase doesn't change
- Feature gate disabled, old unset, phase changes
- Feature gate disabled, old unset, phase doesn't chase
- Feature gate disabled, old set, phase changes
- Feature gate disabled, old set, phase doesn't change

And in all the scenarios, have the input status set the field (so we can see it being overridden)